src/swap/defi/0x/0xGasless.ts

```diff
@@ -181,6 +181,12 @@ export const make0xGaslessPlugin: EdgeCorePluginFactory = opts => {
 
           const orderId: string = apiSwapStatus.transactions[0].hash
 
+          let orderUri: string | undefined
+          const { transactionExplorer } = swapRequest.fromWallet.currencyInfo
+          if (transactionExplorer.includes('%s')) {
```

Comment:
The `if` isn't really needed, because the `.replace` is a no-op if the string isn't there.